next-hardhat/circuits/Nargo.toml

```diff
@@ -1,5 +1,6 @@
 [package]
+name = "noir-starter"
 authors = [""]
 compiler_version = "0.4.1"
```

Comment:
should this be updated?

Reply:
it should but apparently the latest version (that requires that field) is the one that is incompatible with `bb.js` so I'll try to not use it